### PR TITLE
add ci

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,20 +4,23 @@
   "description": "A plugin for minecraft-classic-protocol that adds experimental and bleeding-edge CPE features",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "npm i && protodef-validator data/protocol.json"
```
Comment (rom1504) [inline package.json]: why is npm i needed?

Comment (u9g) [reply package.json]: to install protodef-validator

Comment (u9g) [reply package.json]: oh wait you're right

Comment (u9g) [reply package.json]: done

**Review by rom1504 — APPROVED** [review]

**Pull request merged**